shared/src/main/scala/Dictionary.scala

```diff
@@ -0,0 +1,10 @@
+package vyxal
+object Dictionary:
```

Comment:
```suggestion

private[vyxal] object Dictionary:
```
Not a huge fan of this being able to be modified by everyone. I'd have preferred platform-specific implementations, but if we're not doing that, let's at least make this private to the `vyxal` package.

Reply:
Platform specific implementations was causing too much trouble. A singleton object that's shared between objects with settable dictionaries is much easier.

Reply:
Also, does this mean only vyxal files can modify the dictionary?

Reply:
Yup

Reply:
Who's going to be using Vyxal as a library anyway? Most of the helper functions require context, and that just seems silly when context is vyxal specific.

Reply:
¯\_(ツ)_/¯